_includes/resource.html

```diff
@@ -0,0 +1,15 @@
+<div class="resource">
+    <div>
+        <a href="{{item.website}}" titl="{{item.name}}" class="resource--icon">
+            <img src="/images/{{item.type}}/{{item.icon}}" alt="{{item.name}}"/>
+        </a>
+    </div>
+    <div>
+        <a href="{{item.website}}" title="{{item.name}}’s website" class="resource--title">
```

Comment:
Is this intentionally a fancy quote?
```suggestion
        <a href="{{item.website}}" title="{{item.name}}'s website" class="resource--title">
```

Reply:
can't afford a mac huh?

Reply:
I don't even use one so I have no clue how that got in there.